podman_compose.py

```diff
@@ -2133,6 +2145,7 @@ def _parse_compose_file(self):
         self.x_podman = compose.get("x-podman", {})
 
         args.in_pod_bool = self.resolve_in_pod()
+        args.pod_arg_list = self.resolve_pod_args()
```

Comment:
I think it makes sense to attach `pod_arg_list` to self and then do all processing in `resolve_pod_args()`. That is, include the following in that function too:

```
        if self.global_args.pod_args is None:
            pod_arg_list = None
        else:
            pod_arg_list = shlex.split(self.global_args.pod_args)
```

Reply:
Makes sense. I also restructured the code of `resolve_pod_args` a bit after including the piece you suggested.